recipes/implot/config.yml

```diff
@@ -3,7 +3,3 @@ versions:
     folder: "all"
   "0.13":
     folder: "all"
-  "0.12":
```

Comment:
Is there a reason to drop the old versions? usually we dont unless theres a technical reason

Reply:
It was just an experiment as I was getting those weird "could not find binary package of imgui" errors.

Reply:
I re-added them